test/example/identify-test.ts

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
what about consolidating both examples test in one file?

Reply:
I may combine them later, but I think I'll be adding more tests to `execute-test.ts` so just want to make sure the combined file won't be too big